src/scripts/main.js

```diff
@@ -6,3 +6,44 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+
+const peopleNew = people.map(person => {
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+
+  if (person.sex === 'm') {
+    person.gender = 'Male';
+  }
+
+  if (person.sex === 'f') {
+    person.gender = 'Female';
+  }
+
+  delete person.fatherName;
+  delete person.motherName;
+  delete person.sex;
+  delete person.slug;
+
+  return person;
+});
```

Comment:
You can prepare a new array in a different way. There is no need to mutate existing persons but you can create a new object for every person:
```
const peopleNew = people.map(person => {
  return {
    name: person.name,
    gender: person.sex === 'm' ? 'Male' : 'Female',
    age: person.died - person.born,
    // the same for 'born', 'died', 'century'
  }
)
```

It will be more readable and you will have a shorter code.